op-chain-ops/genesis/config.go

```diff
@@ -249,6 +249,16 @@ func (d *DeployConfig) GetDeployedAddresses(hh *hardhat.Hardhat) error {
 	return nil
 }
 
+// GetDeveloperDeployedAddresses will set the dev addresses on the DeployConfig
+func (d *DeployConfig) GetDeveloperDeployedAddresses() error {
```

Comment:
Maybe call it `InitDeveloperDeployedAddresses` instead? It's not quite a regular getter method

Reply:
I agree that name is better, I used `Get` so it would be similar to the other method for getting deployed addresses, will update

Reply:
I've updated the name